lib/inc/internal/sources/dmi_source.hpp

```diff
@@ -0,0 +1,102 @@
+#pragma once
+
+#include <internal/vm.hpp>
```

Comment:
Are you still using this header here since we lifted parsing the VM name up into the detector logic?